common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -908,9 +926,6 @@
             },
             "ForInStatement"
         ],
-        "no-return-await": [
```

Comment:
This rule seems worth keeping - should we enable it explicitly?

Reply:
Good catch. It's superseded by https://typescript-eslint.io/rules/require-await/, which I will enable.